src/components/layout/layout.js

```diff
@@ -36,6 +36,7 @@ const Layout = ({
     no_login_signup,
     no_live_chat,
     nav_type,
+    CompotentAbove,
```

Comment:
typo here Component, can change the name to Notification? since it works to notify client?

Reply:
nice naming :+1: done.